core/bytearray.go

```diff
@@ -23,6 +25,18 @@ func (b *ByteArray) SetBit(pos int, value bool) {
 	}
 }
 
+// SetBitByArrayPosition sets the bit at the given position to the specified value
```

Comment:
1. Whats the difference between this func `SetBitByArrayPosition` and `func (b *ByteArray) SetBit(pos int, value bool)`
2. Do we need both?

Reply:
So `SetBit` works from right to left, and `SetBitByArrayPosition` works from left to right.
* a.SetBit(3,1) -> 00001000 
* a. SetBitByArrayPosition(3,1) -> 00100000

And same explanation goes for getBit.

We can remove SetBit, I think so... 

And sames for for `GetBit`.

Reply:
Are there any issues with the existing SetBit / GetBit? If not I'd like to avoid writing a new function and remove the old function.
Could you modify the existing functions to address any concerns you have.